resident/resident-service/src/main/java/io/mosip/resident/service/impl/GrievanceServiceImpl.java

```diff
@@ -62,20 +67,22 @@ public ResponseWrapper<Object> getGrievanceTicket(MainRequestDTO<GrievanceReques
         } catch (ApisResourceAccessException e) {
         	logger.error("%s - %s", ResidentErrorCode.GRIEVANCE_TICKET_GENERATION_FAILED.getErrorMessage(), e.getMessage());
             throw new ApisResourceAccessException(ResidentErrorCode.GRIEVANCE_TICKET_GENERATION_FAILED.getErrorCode(), e);
+        } catch (NoSuchAlgorithmException e) {
+            throw new RuntimeException(e);
```

Comment:
Why you are throwing runtime exception please replace with proper Exception

Reply:
corrected